controllers/managedocs_controller.go

```diff
@@ -478,6 +478,11 @@ func (r *ManagedOCSReconciler) reconcilePhases() (reconcile.Result, error) {
 			if err := r.delete(r.managedOCS); err != nil {
 				return ctrl.Result{}, fmt.Errorf("unable to delete managedocs: %v", err)
 			}
+			// Refreshing local managedOCS object after deletion is scheduled
+			// to avoid conflict while updating status
+			if err := r.get(r.managedOCS); err != nil {
+				return ctrl.Result{}, nil
```

Comment:
Why are we ignoring all errors?

Reply:
Should I log the error and pass on to calling function?

Reply:
As discussed offline , if resource not found then logging a warning and for other errors returning the error to the calling function.